salvo/README.md

```diff
@@ -22,6 +22,10 @@ workflow documentation: [ENVOY_DEVELOP_WORKFLOW.md](./ENVOY_DEVELOP_WORKFLOW.md)
 
 The [`install_deps.sh`](./install_deps.sh) script can be used to install any dependencies required
 by Salvo.
+Dependencies installation needs root priviliges to perform. Using below command to make sure proxy(if any) works fine with root context.
+```bash
+sudo http_proxy="example.com:port" https_proxy="example.com:port"  ./install_deps.sh
```

Comment:
Can we also include a command example for running **without** a proxy? That may still be the majority of use cases. Something like:

Dependencies installation needs root privileges to perform. Run the following command to install the required dependencies:

```
sudo ./install_deps.sh
```

If you are using a proxy server, run the following command to download any packages via a proxy server:

```
sudo http_proxy="example.com:port" https_proxy="example.com:port" ./install_deps.sh
```

Reply:
gonna take your helpful advice